src/components/enterprise/dashboard/sidebar/offers/Offer.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+
+import './styles/Offer.scss';
+
+class Offer extends React.Component {
+  renderTitle(usageType, benefitValue) {
+    if (usageType === 'Percentage') {
+      if (benefitValue === 100) {
+        return <>Enroll for free</>;
+      }
+      return <>Get {benefitValue}% off</>;
+    }
+    return <>Get ${benefitValue} off</>;
+  }
+
+  renderFinePrint(redemptionsRemaining, couponEndDate) {
+    let message = `Expires ${moment(couponEndDate).format('MMMM D, YYYY')}.`;
```

Comment:
In the Zeplin mocks, it also shows putting the number of weeks until it expires (e.g., "Expires August 30, 2019 (2 weeks)"

Reply:
I was going off of the comments in ENT-2153 which seemed to indicate we didn't need the weeks part anymore.

Reply:
Ah, missed that comment :) 👍